shell/platform/embedder/embedder.cc

```diff
@@ -222,6 +228,26 @@ static void* DefaultGLProcResolver(const char* name) {
 }
 #endif  // FML_OS_LINUX || FML_OS_WIN
 
+// Auxiliar function used to translate rectangles of type SkIRect to
```

Comment:
These functions can be `static`.

typo: `Auxiliary`.

Reply:
Updated those just now.